frontend/src/layout/lemonade/Breadcrumbs/Breadcrumbs.scss

```diff
@@ -24,11 +24,12 @@
 .Breadcrumbs__breadcrumb {
     display: flex;
     align-items: center;
-    color: var(--primary);
-    font-weight: 500;
+    color: $text_default;
+    font-weight: $medium;
     white-space: pre;
     &--current {
-        font-style: italic;
+        font-weight: bold;
```

Comment:
The final one should be IMO the same weight as the rest. The color is already a strong indicator of the different nature, and bold is quite ugly at a small font size.

Reply:
I'd like to bump this

Reply:
Done